eng/scripts/Create-Apiview-Token-Cadl.ps1

```diff
@@ -19,18 +20,29 @@ function Sparse-Checkout($branchName, $packagePath)
     {
         Remove-Item $sparseCheckoutFile -Force
     }    
-    git sparse-checkout init --cone
-    git sparse-checkout set $packagePath
+    git sparse-checkout init --no-cone
+    git sparse-checkout set $packagePath $globalPackageJsonPath
     git checkout $branchName
 }
 
 function Generate-Apiview-File($packagePath)
 {
+    Copy-Item $globalPackageJsonPath $packagePath/package_global.json
     Write-Host "Generating API review token file from path '$($packagePath)'"
     Push-Location $packagePath
     try
     {
+
+        # Install required packages using project specific package.json          
+        npm install
+        npm list
+        # Install additional/missing packages using gloal package.json
+        Rename-Item package.json package-orig.json

```

Comment:
Is this just for testing or temporary until we get the correct global settings?

Reply:
Or are you trying to enable both sets of package installs?

Reply:
Done.